tester_app/client/webpack.client.base.config.js

```diff
@@ -0,0 +1,58 @@
+// Common client-side webpack configuration used by webpack.hot.config and webpack.rails.config.
+
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = {
+
+  // the project dir
+  context: __dirname,
+  entry: {
+
+    // See use of 'vendor' in the CommonsChunkPlugin inclusion below.
+    vendor: [
+      'babel-core/polyfill',
+      'jquery',
+      'jquery-ujs',
+      'react',
+      'react-dom',
+    ],
```

Comment:
@alexfedoseev @jbhatab @patricesf @robwise 

How do you feel about including lodash here?

I view it as the 3rd major library worth always using for real world apps: 1) react, 2) redux, 3) lodash.